### PR TITLE
Refactor updating scorecard

### PR DESCRIPTION
**Pull request by tunglinn**
Why this refactor?
This simplifies when scorecard updates. 

Before, there were many different cases in which a scorecard would update, like the dropdown changing, the user panning to a different city, or the user clicking on a city. 
Now, the scorecard only updates to the city closes to the center of the screen. So, when the dropdown changes, the map snaps to city, which will also automatically update the scorecard.

### CODE REVIEW

**Review by Eric-Arellano — APPROVED** [review]
Nice! Although technically this isn't a refactor because there is a behavior change for end users. Refactors are when the end behavior is the same and you only change how the code is implemented.

**Comment by tunglinn** [comment]
> Nice! Although technically this isn't a refactor because there is a behavior change for end users. Refactors are when the end behavior is the same and you only change how the code is implemented.

There shouldn't bee a behavior change for the end user. Is there one that you noticed?

**Comment by Eric-Arellano** [comment]
Didn't it change like this?

> Before, there were many different cases in which a scorecard would update, like the dropdown changing, the user panning to a different city, or the user clicking on a city.
> Now, the scorecard only updates to the city closes to the center of the screen. So, when the dropdown changes, the map snaps to city, which will also automatically update the scorecard.

My understanding of the code is that you now call `snapToCity` fewer times than before. But I didn't check this very rigorously.

File: src/js/setUpSite.js [hunk 1]
```diff
@@ -159,18 +159,26 @@ const generateScorecard = (scoreCardEntry) => {
 };
 
 /**
- * Move the map to the city boundaries and set its score card.
+ * Centers view to city.
+ *
+ * @param map: The Leaflet map instance.
+ * @param cityProperties: An object with a `layout` key (Leaflet value) and keys
```
Comment (Eric-Arellano) [inline src/js/setUpSite.js:165]: Outdated

**Comment by tunglinn** [comment]
> Didn't it change like this?
> 
> > Before, there were many different cases in which a scorecard would update, like the dropdown changing, the user panning to a different city, or the user clicking on a city.
> > Now, the scorecard only updates to the city closes to the center of the screen. So, when the dropdown changes, the map snaps to city, which will also automatically update the scorecard.
> 
> My understanding of the code is that you now call `snapToCity` fewer times than before. But I didn't check this very rigorously.

**Comment by tunglinn** [comment]
Yes, but the end user experience doesn't/shouldn't change.

**Pull request merged**